File: Leet_Code/Array/66. Plus One/66. Plus One.java
static int[] plusOne(int[] digits) {
        for (int   i = digits.length-1; i >=0; i--){
            if(digits[i] == 9){
                digits[i] = 0;
            }
            else {
                digits[i] = digits[i] +1;
                return digits;
            }
        }
        int [] new_arr = new int[digits.length+1];
        new_arr[0] = 1;
        return  new_arr;
    }

File: Leet_Code/Array/66. Plus One/66. Plus One.kt
class Solution {
    fun plusOne(digits: IntArray): IntArray {
        for(i in digits.size-1 downTo  0){
        if(digits[i] == 9){
            digits[i] =0
        }
        else {
            digits[i] = digits[i]+1
            return digits
        }
    }
    var new_arr = IntArray(digits.size+1)
    new_arr[0] = 1; 
    return  new_arr;
    }
}
